docs/01-app/01-getting-started/05-fonts.mdx

```diff
@@ -0,0 +1,199 @@
+---
+title: How to use fonts
+nav_title: Fonts
+description: Learn how to use fonts in Next.js
+related:
+  title: API Reference
+  description: See the API Reference for the full feature set of Next.js Font
+  links:
+    - app/api-reference/components/font
+---
+
+The [`next/font`](/docs/app/api-reference/components/font) module automatically optimizes your fonts and removes external network requests for improved privacy and performance.
+
+It includes **built-in self-hosting** for any font file. This means you can optimally load web fonts with no layout shift.
+
+To start using `next/font`, import it from [`next/font/local`](#local-fonts) or [`next/font/google`](#google-fonts), call it as a function with the appropriate options, and set the `className` of the element you want to apply the font to. For example:
```

Comment:
Should this only be done in the root layout? What happens if I do this further down the tree?

Reply:
Since it's a classname, it works similarly to css modules. You can invoked it from anywhere in your app, but it's usually best practice to add it to the root layout so that it applies globally.

I'll add the clarification.